### PR TITLE
Add validation for sites

### PR DESCRIPTION
**Pull request by myokoym**
ref: #168

### CODE REVIEW

File: app/models/site.rb [hunk 1]
```diff
@@ -4,6 +4,9 @@ class Site < ActiveRecord::Base
   has_many :fixed_pages, dependent: :destroy
   has_many :links, dependent: :destroy
 
+  validates :name, presence: true
+  validates :fqdn, presence: true
```
Comment (tricknotes) [inline app/models/site.rb]: How about `uniqueness` ?

Comment (myokoym) [inline app/models/site.rb]: :bulb:

File: test/models/site_test.rb [hunk 2]
```diff
@@ -20,4 +26,32 @@ class SiteTest < ActiveSupport::TestCase
     assert_equal(0, Post.count)
     assert_equal(0, Image.count)
   end
+
+  sub_test_case 'name' do
+    test 'null' do
+      site = Site.create(js_url: "http://example.com/application.js",
+                         css_url: "http://example.com/application.css")
+      assert_false(site.valid?)
+      assert_equal("can't be blank", site.errors[:name][0])
```
Comment (tricknotes) [inline test/models/site_test.rb]: If site has some other errors, this test may failed :broken_heart: 
Can you use other assertion what something like [`assert_includes`](http://docs.ruby-lang.org/ja/2.1.0/class/MiniTest=3a=3aAssertions.html#I_ASSERT_INCLUDES)?

File: test/models/site_test.rb [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 class SiteTest < ActiveSupport::TestCase
   setup do
+    @default_locale = I18n.locale
+    I18n.locale = :en
```
Comment (tricknotes) [inline test/models/site_test.rb]: I think we want to test against `ja`.

File: test/models/site_test.rb (hunk 2 above)
Comment (myokoym) [inline test/models/site_test.rb]: Thanks! We can use `assert_includes` but I think it should be failed if site has some other errors that are unexpected. However, we should get some other errors easy. I'll compare remain array.

Comment (tricknotes) [inline test/models/site_test.rb]: reasonable :+1:

File: test/models/site_test.rb (hunk 3 above)
Comment (myokoym) [inline test/models/site_test.rb]: I'll use `ja` for the following points:
- The product is almost used by Japanese.
- If we use `ja`, we can test against localization configuration of error messages.

**Pull request merged**